22-discovery.ks

```diff
@@ -57,19 +57,14 @@ sed -i '/\[Service\]/a PermissionsStartOnly=true' /usr/lib/systemd/system/forema
 cat >/etc/foreman-proxy/settings.yml <<'CFG'
 ---
 :settings_directory: /etc/foreman-proxy/settings.d
-
 # certificate is generated by /usr/bin/generate-proxy-cert
 :ssl_certificate: /etc/foreman-proxy/cert.pem
 :ssl_ca_file: /etc/foreman-proxy/cert.pem
 :ssl_private_key: /etc/foreman-proxy/key.pem
-
 :daemon: true
 :http_port: 8448
 :https_port: 8443
-
-# SYSLOG cannot be used, see: http://projects.theforeman.org/issues/11623
-# :log_file: SYSLOG
-:log_file: /tmp/proxy.log
+:log_file: SYSLOG
```

Comment:
It's this one. Let me try it now... https://github.com/theforeman/smart-proxy/pull/370